core/inc/knoting/window.h

```diff
@@ -26,6 +41,12 @@ class Window {
     int m_height;
     std::string m_title;
 
+    double m_deltaTime = 0;
```

Comment:
As we discussed, do we want to track time inside the Window class, or is this a temporary solution?

Reply:
Time submodule will be added later on that will handle timesteps